.gitignore

```diff
@@ -9,6 +9,8 @@ __pycache__
 # Ignore .c files by default to avoid including generated code. If you want to
 # add a non-generated .c extension, use `git add -f filename.c`.
 *.c
+# ... but don't ignore this file
+!tests/ufunc_example/_module2.c
```

Comment:
I don't think this is necessary - once you have added a file, it doesn't get ignored anyway. This is just for whether things like `git status` will list `*.c` files.

Reply:
OK, reverted.